scripts/apply_committable_suggestions.py

```
@@ -0,0 +1,258 @@
+        return None
+
+    return Suggestion(
+        comment_id=comment["id"],
+        path=path,
+        start_line=int(start_line),
+        end_line=int(line),
+        new_text=match.group("body").rstrip("\n"),
+    )
+
+
+def _apply_one(svg_path: Path, sug: Suggestion) -> bool:
```

Ruff (0.15.12)

[warning] Unused function argument: `svg_path`

(ARG001)